### PR TITLE
Scan for updates to new images

### PR DESCRIPTION
**Pull request by kfox1111**
Fixes: https://github.com/spiffe/helm-charts/issues/465

Also adds support for registries that are blank (IE, docker.io) and fixes a bug with the pip install that installs to the wrong place.

### CODE REVIEW

File: .github/scripts/update-tags.sh [hunk 1]
```diff
@@ -46,18 +46,21 @@ jq -r '. | keys[]' "$IMAGEJSON" | while read -r CHART; do
     REGISTRY=$(yq e ".${QUERY}.registry" "$VALUES")
     REPOSITORY=$(yq e ".${QUERY}.repository" "$VALUES")
     VERSION=$(yq e ".${QUERY}.tag" "$VALUES")
```
Comment (kfox1111) [inline .github/scripts/update-tags.sh:48]: These changes allow blank Registries (IE, docker.io). First time we've seen one in the chart.

File: .github/scripts/update-tags.sh [hunk 2]
```diff
@@ -46,18 +46,21 @@ jq -r '. | keys[]' "$IMAGEJSON" | while read -r CHART; do
...
+    if [[ "$REGISTRY" != "" ]]; then
+      REGISTRY="$REGISTRY/"
```
Comment (marcofranssen) [inline .github/scripts/update-tags.sh:50]: 👍

**Review by marcofranssen — APPROVED** [review]
LGTM 🚀

**Review by faisal-memon — APPROVED** [review]
Thanks @kfox1111 for all the automation work.

**Pull request merged**